packages/gasket-plugin-express/lib/create-servers.js

```diff
@@ -131,9 +129,16 @@ module.exports = async function createServers(gasket, serverOpts) {
   await gasket.exec('express', app);
 
   if (routes) {
-    const files = await glob(`${ routes }.js`, { cwd: root });
-    for (const file of files) {
-      require(path.join(root, file))(app);
+    for (const route of routes) {
+      if (typeof route !== 'function') {
+        throw new Error('Route must be a function');
+      }
+
+      if (route.constructor.name === 'AsyncFunction') {
+        await route(app);
+      } else {
+        route(app);
+      }
```

Comment:
Should be ok just to await all these for simplicity
```suggestion
      await route(app);
```